### ISSUE TITLE
Issues Compiling Sass??? Maybe?

### ISSUE THREAD

**Issue by xxxcodeingxxx** [JeremyEnglert/JointsWP#415]
I am not sure why but when I make changes to SASS files sometimes the changes will take place on browsersync reload and other times I have to manually refresh the browser. I am using latest Chrome browser and there is no caching on my WordPress site. I don't get it?

**Comment by anaelleltd** [comment]
I am experiencing the same problem. 
I'm still troubleshooting to see what the real issue is, since it does not come from NPM.

I tried installing node-sass independently, but the error message persists.

**Comment by anaelleltd** [comment]
This is what I did to get it working:
1. manually delete the node modules folder from the theme folder.
2. run the CLI and enter npm install node-sass --force (this forces the installation of node-sass)
3. enter npm install (this installs the remaining node module files)
4. enter npm rebuild (this patches all installations together)
5. enter npm uninstall (I believe this cleared the cache??)
6. enter npm install-test (this attempts to run the build and displays error with .scss or .js files)
Basically, at this point, the npm build is working without throwing a "missing gulp sass" error.
7. enter npm run build 
8. refresh browser to see updates/changes

**Comment by xxxcodeingxxx** [comment]
> This is what I did to get it working:
> 
> 1. manually delete the node modules folder from the theme folder.
> 2. run the CLI and enter npm install node-sass --force (this forces the installation of node-sass)
> 3. enter npm install (this installs the remaining node module files)
> 4. enter npm rebuild (this patches all installations together)
> 5. enter npm uninstall (I believe this cleared the cache??)
> 6. enter npm install-test (this attempts to run the build and displays error with .scss or .js files)
>    Basically, at this point, the npm build is working without throwing a "missing gulp sass" error.
> 7. enter npm run build
> 8. refresh browser to see updates/changes

Shouldn't Gulp refresh the browser for you though when you make changes? It's been a while since I've used Gulp, I don't remember having to refresh the browser every time you make a change.

**Comment by garretthyder** [comment]
It only refreshes the browser when you are 'watch'ing it;
https://github.com/JeremyEnglert/JointsWP#watching-for-changes
*edit was just to provide the right link
